susemanager/src/mgr_bootstrap_data.py

```diff
@@ -681,4 +761,14 @@
         'BASECHANNEL' : 'centos7-x86_64', 'PKGLIST' : RES7,
         'DEST' : '/srv/www/htdocs/pub/repositories/centos/7/bootstrap/'
     },
+    'ubuntu-16.04-amd64' : {
+        'BASECHANNEL' : 'ubuntu-16.04-amd64-main', 'PKGLIST' : PKGLISTUBUNTU1604,
```

Comment:
Wrong base channel name. The name we use now is: `ubuntu-16.04-pool-amd64`

Reply:
Changed base channel to `ubuntu-16.04-pool-amd64`